bin/codeclimate-checkstyle

```diff
@@ -13,4 +13,8 @@ else
   engine_config = {}
 end
 
-CC::Engine::Checkstyle.new("/code", engine_config, STDOUT).run
+begin
+  CC::Engine::Checkstyle.new("/code", engine_config, STDOUT).run
+rescue => e
```

Comment:
Getting stack traces here from genuine bugs can be really helpful for debugging. Might be better to just leave off the rescue & let the full exception details print.

If you don't want this particular error case to cause that, I think you'd be better of doing an explicit `exit 1` instead of raising.